.github/workflows/cypress.yml

```diff
@@ -33,36 +42,51 @@ jobs:
       - name: Install bootstrapper dependencies
         run: pipenv install --dev --deploy
       - run: pipenv run cookiecutter . --no-input -f
+      - uses: actions/upload-artifact@v3
+        with:
+          name: my_project
+          path: my_project/
+          retention-days: 1
+    outputs:
+      BASE_URL: ${{ env.BASE_URL }}
+  Chrome:
+    needs: Setup
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: actions/download-artifact@v3
+        with:
+          name: my_project
+          path: my_project/
       - name: Install frontend dependencies
         env:
           NPM_CONFIG_PRODUCTION: false
         working-directory: ./my_project/client
         run: npm install
       - name: Run against ${{ needs.Setup.outputs.BASE_URL }}
-        uses: cypress-io/github-action@v2
+        uses: cypress-io/github-action@v4
         with:
           working-directory: my_project/client
           browser: chrome
-          headless: true
```

Comment:
no longer a thing. It's headless by default. Use `headed` to actually have a browser available.